system-int-test/src/cmr/system_int_test/utils/variable_util.clj

```diff
@@ -0,0 +1,328 @@
+(ns cmr.system-int-test.utils.variable-util
+  "This contains utilities for testing variable"
+  (:require
+   [clojure.string :as string]
+   [cmr.acl.core :as acl]
+   [clojure.test :refer [is]]
+   [cmr.common.mime-types :as mt]
+   [cmr.common.util :as util]
+   [cmr.mock-echo.client.echo-util :as e]
+   [cmr.system-int-test.data2.core :as d]
+   [cmr.system-int-test.system :as s]
+   [cmr.system-int-test.utils.index-util :as index]
+   [cmr.system-int-test.utils.metadata-db-util :as mdb]
+   [cmr.system-int-test.utils.search-util :as search]
+   [cmr.transmit.echo.tokens :as tokens]
+   [cmr.transmit.variable :as transmit-variable]))
+
+(defn- get-system-ingest-update-acls
```

Comment:
A lot of these functions are access-control related not specific to variables at all. It would be good to have an access-control utility type namespace to put these in.